dali/operators/reader/coco_reader_op.cc

```diff
@@ -156,8 +158,26 @@ preprocessed COCO annotations.)code",
     })
   .AddParent("LoaderBase");
 
+
+// Deprecated alias
 DALI_REGISTER_OPERATOR(COCOReader, COCOReader, CPU);
 
+DALI_SCHEMA(COCOReader)
+    .NumInput(0)
+    .NumOutput(3)
+    .DocStr("Legacy alias for :meth:`readers.coco`.")
+    .AdditionalOutputsFn([](const OpSpec &spec) {
+      return OutPolygonMasksEnabled(spec) * 2 + OutPixelwiseMasksEnabled(spec) +
+             OutImageIdsEnabled(spec);
+    })
```

Comment:
Nitpick: this function is the same as above, but formatting obfuscates it - perhaps, though, this could be factored out, like in `Caffe2Reader`?

Reply:
Sorry, missed that one.